scipy/sparse/construct.py

```diff
@@ -791,5 +791,20 @@ def rand(m, n, density=0.01, format="coo", dtype=None, random_state=None):
     -----
     Only float types are supported for now.
 
+    See Also
+    --------
+    scipy.sparse.random : Equivalent function when passing `data_rvs=None`
```

Comment:
I meant something like:
```
scipy.sparse.random : Similar function that allows a user-specified random data source.
```

Or maybe stronger language than that, but I'm not sure.